tests/tests_suite_test.go

```diff
@@ -31,7 +31,7 @@ const (
 	// TODO - maybe randomize namespace
 	testNamespace             = "ssp-operator-functests"
 	commonTemplatesTestNS     = "ssp-operator-functests-templates"
-	timeout                   = 180 * time.Second
+	timeout                   = 90 * time.Second
```

Comment:
I feel 90 might be little less, because some test machines can be very slow at times

Reply:
Changed back. Previously the 180s was problematic, because all tests were timing out in `BeforeEach`.
We have 57 tests, so it took 2 hours and 51 minutes until all tests finished.

With this change, there is a short circuit when waiting for deployment, so it will fail fast.